vllm/executor/uniproc_executor.py

```diff
@@ -34,13 +34,13 @@ def _init_executor(self) -> None:
         if len(device_info) > 1:
             local_rank = int(device_info[1])
         rank = 0
+        is_driver_worker = True
         kwargs = dict(
             vllm_config=self.vllm_config,
             local_rank=local_rank,
             rank=rank,
             distributed_init_method=distributed_init_method,
-            is_driver_worker=(not self.parallel_config)
-            or (rank % self.parallel_config.tensor_parallel_size == 0),
+            is_driver_worker=is_driver_worker,
```

Comment:
Thanks @lifuhuang, but why have separate variable?
```suggestion
            is_driver_worker=True,
```

Reply:
Thank you @njhill for reviewing!

The only reason is to be consistent with other locations in the file, e.g. [uniproc_executor.py#L112](https://github.com/vllm-project/vllm/blob/183dad7a85487dbb351c43c11d2180a6108d5448/vllm/executor/uniproc_executor.py#L112). Seems to be the preferred coding style of @youkaichao ? :)

Reply:
either looks good to me.